Adding Pipenv for our project and install Django

shayakrach opened the pull request.

yuvalk commented:
can you please make your commit msgs more meaningful?
try to explain the "why"
also you have 2 commits touching the same area, please squash these

shayakrach committed:
Adding a Django development environment

- Setup [Pipenv][1] to install and manage Django and other Python
dependencies
- Bootstrapped an empty [Django][2] application

assafg95 commented:
Went over the Commit and looks like everything is ok. I understand the commit message perfectly and understand why it is added.  Looks ready to be merged
